### ISSUE TITLE
Export to different extensions

### ISSUE THREAD

**Issue by trinanda** [cuducos/alchemydumps#18]
Can I backup the file in different extensions..?, such as **.zip** or plaint text..?

**Comment by cuducos** [comment]
No, at this time, you cannot. But I agree this would be a nice feature, indeed. I'll let this issue open an invitation (and reminder) for future features.
